src/tools/linkchecker/Cargo.toml

```diff
@@ -10,3 +10,4 @@ path = "main.rs"
 [dependencies]
 regex = "1"
 html5ever = "0.29.0"
+clap = { version = "4.5.40", features = ["derive"] }
```

Comment:
I'm a little uneasy adding this because this tool intentionally tries to keep its dependencies down to a minimum because it is used in many repositories.

Are there maybe some alternatives here? Do this really need a full feature CLI parser? At a minimum, can this drop the default features? Perhaps use the builder API instead of derive?